backend/heartcode/src/main/java/com/itvitae/heartcode/user/UserController.java

```diff
@@ -33,8 +33,10 @@ public AuthTokenDTO register(@RequestBody RegisterDTO registerDTO) {
     if (registerDTO.alias() == null || registerDTO.alias().isBlank()) {
       throw new BadRequestException("alias is required");
     }
-    if (registerDTO.password() == null || registerDTO.password().isBlank()) {
-      throw new BadRequestException("password is required");
+    if (registerDTO.password() == null || registerDTO.password().isBlank() || registerDTO.password().length() < 7) {
```

Comment:
is er een specifieke reden dat password length niet wordt gecheckt door 'userService.isValidPassword'?